catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -154,13 +164,53 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       query.setFacetMinCount(textFacetProp.getMinFacetCount());
     }
 
-    long totalHits;
-    List<Result> results = new ArrayList<>();
-    Map<String, Serializable> responseProps = new HashMap<>();
+    Serializable suggestQueryProp = request.getPropertyValue(SUGGESTION_REQUEST_KEY);
+
+    if (suggestQueryProp instanceof String) {
+      query = new SolrQuery();
+      query.setRequestHandler("/suggest");
+      query.setParam(SUGGEST_Q, (String) suggestQueryProp);
+      query.setParam(SUGGEST_CONTEXT_FILTER_QUERY, GAZETTEER_METACARD_TAG);
```

Comment:
❗️ I don't think this class should have to know about the Gazetteer or its tag.  You might want to add a Suggest Request Context property instead of hardcoding this here.